packages/commons/src/config/EnvironmentVariablesService.ts

```diff
@@ -46,7 +52,7 @@ class EnvironmentVariablesService implements ConfigServiceInterface {
    * Get the value of the `POWERTOOLS_SERVICE_NAME` environment variable.
    */
   public getServiceName(): string {
-    return this.get(this.serviceNameVariable);
+    return getServiceName();
   }
```

Comment:
~~This function is redundant now. I see that we reuse everything from the utils now. Why do wee need this service that only facades the utils implementation?~~

Edit: nvm, just reread the PR and I agree that we migrate it at some other point. We can keep it for now.